tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -213,6 +211,10 @@ Standard-Section: "ASTs" TopLevelStat*
                   OPEN                                                             -- an open class
                   Annotation
 
+  Variance      = SEALED                                                           -- invariant
```

Comment:
The implementation actually uses the STABLE flag:
```suggestion
  Variance      = STABLE                                                           -- invariant
```